packages/components/stories/List.js

```diff
@@ -790,5 +800,16 @@ storiesOf('List', module)
 			<IconsProvider defaultIcons={icons} />
 			<List {...getPropsFor('tile')} />
 		</div>
+	))
+	.add('DEPRECATED - list with i18n', () => (
```

Comment:
Please turn that into a non deprecated story please. The translation is only on toolbar, you can make the same story with Virtualized story